var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -89,9 +91,26 @@ class Trilinos(CMakePackage):
             description='Builds a debug version of the libraries')
     variant('boost',        default=True,  description='Compile with Boost')
     variant('tpetra',       default=True,  description='Compile with Tpetra')
-    variant('exodus',       default=False, description='Compile with Exodus from SEACAS')
-
-    variant('dtk',          default=False, description='Enable DataTransferKit')
+    variant('epetra',       default=True,  description='Compile with Epetra')
+    variant('exodus',       default=False,
+            description='Compile with Exodus from SEACAS')
+    variant('pnetcdf',      default=False,
+            description='Compile with parallel-netcdf')
+    variant('zlib',         default=True,  description='Compile with zlib')
+    variant('stk',          default=False, description='Compile with STK')
+    variant('belos',        default=False, description='Compile with Belos')
+    variant('zoltan',       default=False, description='Compile with Zoltan')
+    variant('amesos',       default=False, description='Compile with Amesos')
+    variant('ifpack',       default=False, description='Compile with Ifpack')
+    variant('muelu',        default=False, description='Compile with Muelu')
+    variant('fortran',      default=True,
+            description='Compile with Fortran support')
```

Comment:
keep on the same line to have consistent indentation, same above for `pnetcdf` and others.